crates/viewer/re_data_ui/src/blob.rs

```diff
@@ -341,3 +336,71 @@ fn show_video_blob_info(
         }
     }
 }
+
+fn decoded_frame_ui(
```

Comment:
maybe it's time to move these functions to a sub-module, ie. `re_data_ui/src/blob/video.rs` 🤔